new-pipeline/src/dist.js

```diff
@@ -914,6 +918,28 @@ function displayHistograms(histogramsList, dates, useTable, cumulative, trim, in
     if (histogramsList.length === 1 && histogramsList[0].histograms.length ===
       1) { // Only show one set of axes
       var histogram = histogramsList[0].histograms[0];
+      if (sortByValue) {
+        let sorting = [];
+        for (let index = 0; index < histogram.buckets.length; index++) {
+          sorting.push([histogram.values[index], histogram.buckets[index]]);
+        }
+        sorting.sort((a,b) => {
+          rv = b[0] - a[0];
+          if (rv != 0) {
+            return rv;
+          }
+          if (a[1] < b[1]) {
+            return -1;
+          }
+          if (a[1] > b[1]) {
+            return 1;
+          }
+          return 0;
```

Comment:
If you think it's not too difficult to read/maintain we could condense to:

```js
sorting.sort((a, b) => b[0] - a[0] || a[1].localeCompare(b[1]));
```

Reply:
Changed to this, which removes the previous two nits. 🙂